templates/core/_init_tabs.html

```diff
@@ -2,11 +2,20 @@
 {# tab id must be {{ tabs }} (use {% with...%}) #}
 {% load i18n %}
 <script>
-
+const getCookieByName = (cookieName) => document.cookie.split('; ')
+    .find((cookie) => cookie.split('=')[0] === cookieName)?.split('=')[1] || null;
 
 $(document).ready(function() {
     tabprefix = "goto_";
 
+    // detect mobile devices using screen size
+    const deviceType = Math.min(window.screen.width, window.screen.height) < 768 ? 'mobile' : 'desktop';
+    const deviceTypeCookie = getCookieByName('deviceType')
+    // update the device type if it has changed
+    if (deviceTypeCookie === null || deviceTypeCookie !== deviceType) {
+        document.cookie = 'deviceType=' + deviceType;
```

Comment:
it leads to the following warning:
`Cookie “deviceType” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite`

Reply:
Solved with `; SameSite=Strict`